pkg/AnewLibrary/Fields/atom-manifest.js

```diff
@@ -19,6 +19,13 @@ export const Fields= {
     },
     outputs: {
       value: 'Boolean'
+    },
+    state: {
```

Comment:
cool, i like it (yes, you mentioned it to me before, but now I can see these dots connect!) 🥳